test/siwapp/items_test.exs

```diff
@@ -0,0 +1,45 @@
+defmodule Siwapp.ItemsTest do
+  use Siwapp.DataCase
+
+  alias Siwapp.Commons.Tax
+  alias Siwapp.Invoices.Item
+
+  describe "base_amount(item) Result of the multiply between the quantity and the unitary cost of an item" do
```

Comment:
try to be much more concise in the descriptions and strings of the tests. It's very cumbersome to read all this, being things so simple.